vm/devices/user_driver/src/memory.rs

```diff
@@ -15,7 +15,7 @@ pub const PAGE_SIZE64: u64 = PAGE_SIZE as u64;
 
 /// A mapped buffer that can be accessed by the host or the device.
 ///
-/// # Safety
+/// SAFETY:
```

Comment:
No, `# Safety` is correct within doc comments.

Reply:
ok, will revert